Makefile

```diff
@@ -818,10 +818,10 @@ strerr.h
 	./compile maildir.c
 
 maildir2mbox: \
-load maildir2mbox.o maildir.o prioq.o now.o myctime.o gfrom.o lock.a \
```

Comment:
I don’t think this branch wants to include the `now()` change — rebase and re-push this one?

Reply:
That's just the glitch of the GitHub Web UI as I rebased it on the remove-now branch before merging that to master, but at the same commit.